libsplinter/src/biome/credentials/store/diesel/mod.rs

```diff
@@ -14,8 +14,6 @@
 
 pub(in crate::biome) mod models;
 mod operations;
-#[cfg(feature = "postgres")]
```

Comment:
Typos in commit msg: s/into in/into/ and s/hourse/house/

Reply:
Added "Part of centralizing migrations" to all commit messages